internal/certdb/certdb_test.go

```diff
@@ -0,0 +1,117 @@
+package certdb_test
+
+import (
+	"log"
+	"strings"
+	"testing"
+
+	"github.com/canonical/gocert/internal/certdb"
+)
+
+func TestConnect(t *testing.T) {
+	db := new(certdb.CertificateRequests)
+	defer db.Disconnect()
+	if err := db.Connect(":memory:", "CertificateReqs"); err != nil {
+		t.Fatalf("Can't connect to SQLite: %s", err)
```

Comment:
I feel like there's no point in continuing a test if one fails, since we'll have to address it regardless and the tests complete in less than a second.